src/riddle/useCases/solveRiddle/createSolveRiddleService.ts

```diff
@@ -26,17 +26,8 @@ export const createSolveRiddleModel = ({
         return '';
     };
 
-    const getAnswerOptionClassName = (answerId: string) => ({
-        'cursor-pointer': !selected,
-        'border-blue-500': !correct,
-        "border-green-700 text-green-900 before:content-['✓']":
-            selected === answerId && correct && correct === answerId,
-        "border-red-700 text-red-800  before:content-['✗']":
-            selected === answerId && correct && correct !== answerId,
```

Comment:
It makes sense to have this logic here within the model, since we can assert it via unit tests. But CSS classes shall go closer to the JSX output I refer in